<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlottingDialog1</class>
 <widget class="QDialog" name="PlottingDialog1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建曲线</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 全局样式：白底黑字，圆角，现代化风格 */
QDialog, QWidget {
    background-color: white;
    color: #333333;
    font-family: &quot;Microsoft YaHei&quot;, &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}
QLabel {
    color: #333333;
}
QLineEdit, QComboBox, QSpinBox {
    color: #333333;
    background-color: #fdfdfd;
    border: 1px solid #c0c0c0;
    border-radius: 4px;
    padding: 4px;
    min-height: 20px;
}
QLineEdit:focus, QComboBox:focus, QSpinBox:focus {
    border: 1px solid #0078d7;
}
QGroupBox {
    border: 1px solid #e0e0e0;
    margin-top: 20px;
    padding-top: 15px;
    border-radius: 4px;
    background-color: #fafafa;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
    color: #0078d7;
    font-weight: bold;
}
QPushButton {
    color: #333333;
    background-color: #f0f0f0;
    border: 1px solid #c0c0c0;
    border-radius: 4px;
    padding: 6px 16px;
}
QPushButton:hover {
    background-color: #e0e0e0;
    border-color: #0078d7;
}
QPushButton:pressed {
    background-color: #d0d0d0;
}
QCheckBox {
    color: #333333;
    spacing: 8px;
}
QCheckBox::indicator {
    width: 18px;
    height: 18px;
    border: 1px solid #c0c0c0;
    border-radius: 3px;
    background: white;
}
QCheckBox::indicator:checked {
    background-color: #0078d7;
    border: 1px solid #0078d7;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_Data">
     <property name="title">
      <string>数据设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_Data">
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_Name">
        <property name="text">
         <string>曲线名称:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_Name"/>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="check_ShowSource">
        <property name="text">
         <string>显示数据来源</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_Source">
        <property name="text">
         <string>数据文件:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboFileSelect"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_X">
        <property name="text">
         <string>X轴数据:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="combo_XCol"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_Y">
        <property name="text">
         <string>Y轴数据:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="combo_YCol"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Style">
     <property name="title">
      <string>样式设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_Style">
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_PointShape">
        <property name="text">
         <string>点形状:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="combo_PointShape"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_PointColor">
        <property name="text">
         <string>点颜色:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="combo_PointColor">
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_LineStyle">
        <property name="text">
         <string>线类型:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="combo_LineStyle"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_LineColor">
        <property name="text">
         <string>线颜色:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="combo_LineColor">
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_LineWidth">
        <property name="text">
         <string>线宽度:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spin_LineWidth">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="check_NewWindow">
     <property name="text">
      <string>在新建窗口中显示</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PlottingDialog1</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PlottingDialog1</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
